### ISSUE TITLE
Link hints with 'f' open in a new tab

### ISSUE THREAD

**Issue by npit** [1995eaton/chromium-vim#653]
See title. `f` and `F` behave in the same way.

**Comment by zyf0330** [comment]
Some links have default behavior that opens at new tab when click.

**Comment by npit** [comment]
This behaviour is observed in *all* links, though.
Using chromium Version 70.0.3538.110 on Arch.

**Comment by zyf0330** [comment]
![image](https://user-images.githubusercontent.com/11958657/49123597-2ca8c480-f2f4-11e8-9053-87fbadf5312b.png)
When you use f to this title link, which place does new page be opened, this tab or new tab?

**Comment by npit** [comment]
Both `f` and `F` open the link in a new tab.

**Comment by npit** [comment]
Yeah, it behaves the same. Thanks for the help - I'll update this if I figure it out.

**Comment by khipukamayuq** [comment]
@npit any luck on solution? @zyf0330 I've run into the same issue on Ubuntu 18.04

**Comment by npit** [comment]
@khipukamayuq No, I haven't ran across anything.

**Comment by zyf0330** [comment]
Sorry, I am on Ubuntu 18.10 and have no this problem.

**Comment by caikaidi** [comment]
Same problem on macos 10.14.5

**Comment by ivanjermakov** [comment]
Same problem on on Linux Mint 19.2 Tina. Any progress?

**Comment by caikaidi** [comment]
> Same problem on on Linux Mint 19.2 Tina. Any progress?

yep, 'f' and lowercase the folowing 1 or 2 letters.
